hy/compiler.py

```diff
@@ -745,6 +745,12 @@ def _render_quoted_form(self, form, level):
             return imports, HyExpression([HySymbol(name),
                                           HyString(form)]).replace(form), False
 
+        elif isinstance(form, HyString):
+            l = [HySymbol(name), form]
+            if form.brackets is not None:
+                l.extend([HyKeyword(":brackets"), form.brackets])
+            return imports, HyExpression(l).replace(form), False
```

Comment:
This is minor, but I'd prefer to avoid single-character names that can be confused with other characters in Python, like `lI1` or `0O`. These are distinguishable in a good programmer font, but you might have to squint. This is the kind of thing flake8 should be able to catch.